newdle/client/src/components/WelcomeHeader.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './WelcomeHeader.module.scss';
+
+const WelcomeHeader = () => (
+  <div className={styles.box}>
+    <h3>Welcome to newdle!</h3>
+    <p>
+      <strong>newdle</strong> is a collective meeting scheduling application. <br />
```

Comment:
is the space before `<br />` intentional?

Reply:
Nope.